src/domain/estimations/estimations.repository.interface.ts

```diff
@@ -0,0 +1,12 @@
+import { EstimationRequest } from './entities/estimation-request.entity';
+import { Estimation } from './entities/estimation.entity';
+
+export const IEstimationsRepository = Symbol('IEstimationsRepository');
+
+export interface IEstimationsRepository {
+  createEstimation(
```

Comment:
Shouldn't this be `getEstimation`? We are trying to get an `Estimation` given the `EstimationRequest`

Reply:
Yes, I agree. Maybe I was influenced by the fact this is using a POST endpoint to talk to the Transaction Service, but of course that's an implementation detail. `getEstimation` is better, I'll change it 👍🏻

Reply:
[cc1b105](https://github.com/5afe/safe-client-gateway-nest/pull/269/commits/cc1b10550ff69e6516e4a0956cc3820a7d6f5c43)